examples/Microsoft.CognitiveServices_accounts_connections@2025-06-01/main.tf

```diff
@@ -0,0 +1,307 @@
+terraform {
+  required_providers {
+    azapi = {
+      source = "Azure/azapi"
+    }
+  }
+}
+
+provider "azapi" {
+  skip_provider_registration = false
+}
+
+variable "resource_name" {
+  type    = string
+  default = "acctest0001"
+}
+
+variable "location" {
+  type    = string
+  default = "westeurope"
+}
+
+data "azapi_client_config" "current" {}
+
+resource "azapi_resource" "resourceGroup" {
+  type     = "Microsoft.Resources/resourceGroups@2020-06-01"
+  name     = var.resource_name
+  location = var.location
+}
+
+resource "azapi_resource" "userAssignedIdentity" {
+  type                   = "Microsoft.ManagedIdentity/userAssignedIdentities@2024-11-30"
+  name                   = var.resource_name
+  location               = var.location
+  parent_id              = azapi_resource.resourceGroup.id
+  response_export_values = ["*"]
+}
+
+resource "azapi_resource" "account" {
+  type      = "Microsoft.CognitiveServices/accounts@2025-06-01"
+  parent_id = azapi_resource.resourceGroup.id
+  name      = var.resource_name
+  location  = var.location
+
+  body = {
+    identity = {
```

Comment:
Please use the identity block in the root level, https://registry.terraform.io/providers/azure/azapi/latest/docs/resources/resource#nestedblock--identity

Reply:
fixed